.buildkite/pipelines/serverless.yml

```diff
@@ -66,11 +66,77 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 40
+    parallelism: 12
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
+    artifact_paths:
+      - "target/kibana-security-serverless/**/*"
+
+  - command: .buildkite/scripts/steps/functional/security_serverless.sh
+    label: 'Explore - Security Solution Cypress Tests'
+    agents:
+      queue: n2-4-spot
+    depends_on: build
+    timeout_in_minutes: 40
+    parallelism: 12
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
+    artifact_paths:
+      - "target/kibana-security-serverless/**/*"
+
+  - command: .buildkite/scripts/steps/functional/security_serverless.sh
+    label: 'Serverless Security Cypress Tests'
+    agents:
+      queue: n2-4-spot
+    depends_on: build
+    timeout_in_minutes: 40
+    parallelism: 12
     retry:
       automatic:
-        - exit_status: '-1'
-          limit: 3
         - exit_status: '*'
           limit: 1
     artifact_paths:
-      - "target/kibana-security-serverless/**/*"
\ No newline at end of file
+      - "target/kibana-security-serverless/**/*"
+
+  - command: .buildkite/scripts/steps/functional/security_serverless_explore.sh
+    label: 'Explore - Security Solution Cypress Tests'
+    agents:
+      queue: n2-4-spot
+    depends_on: build
+    timeout_in_minutes: 60
+    parallelism: 4
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
+    artifact_paths:
+      - "target/kibana-security-serverless/**/*"
+
+  - command: .buildkite/scripts/steps/functional/security_serverless_investigations.sh
+    label: 'Investigations - Security Solution Cypress Tests'
+    agents:
+      queue: n2-4-spot
+    depends_on: build
+    timeout_in_minutes: 120
+    parallelism: 6
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
+    artifact_paths:
+      - "target/kibana-security-serverless/**/*"
+
+  - command: .buildkite/scripts/steps/functional/security_serverless_burn.sh
```

Comment:
Is this expected in an on-merge pipeline?

Reply:
great catch, Thank you @jbudz 🙇